Makefile

```diff
@@ -0,0 +1,34 @@
+.DEFAULT_GOAL := all
+
+.PHONY: slim
+slim:
```

Comment:
Make these targets simpler and just run slim/appinspect/pylint directly instead of building a Docker image and running it.

Reply:
> Make these targets simpler and just run slim/appinspect/pylint directly instead of building a Docker image and running it.

So assume that slim/appinspect/pylint are pre-installed locally?

Reply:
Yes, as far as the Makefile is concerned.